tests/test_cli_compile.py

```diff
@@ -3447,6 +3447,42 @@ def test_allow_in_config_pip_sync_option(pip_conf, runner, tmp_path, make_config
     assert "Using pip-tools configuration defaults found" in out.stderr
 
 
+def test_use_src_files_from_config_if_option_is_not_specified_from_cli(
+    pip_conf, runner, tmp_path, make_config_file
+):
+    foo_in = tmp_path / "foo.in"
+    req_in = tmp_path / "requirements.in"
+
+    config_file = make_config_file("src-files", [foo_in.as_posix()])
+
+    req_in.write_text("small-fake-a==0.1", encoding="utf-8")
+    foo_in.write_text("small-fake-b==0.1", encoding="utf-8")
+
+    out = runner.invoke(cli, ["--config", config_file.as_posix()])
+
+    assert out.exit_code == 0, out
+    assert "small-fake-b" in out.stderr
+    assert "small-fake-a" not in out.stderr
```

Comment:
Could you please check the whole output in `out.stderr` (here and in the other test)? See `test_not_specified_input_file` as a reference.

Reply:
I tried to stick to what other tests did, and also this is enough, since I am showing that one of the two requirements file is ignored